meta-lmp-bsp/conf/machine/imx8mm-lpddr4-evk-ebbr.conf

```
@@ -0,0 +1,11 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX 8M Mini EVK with LPDDR4 and EBBR
+#@SOC: i.MX8MM
+#@DESCRIPTION: Machine configuration for NXP i.MX 8M Mini Evaluation Kit with LPDDR4
+#              following the EBBR requirements (UEFI)
+
+require conf/machine/imx8mm-lpddr4-evk.conf
+
+KMACHINE = "imx8mmevk"
+MACHINE_FEATURES += "efi ebbr"
```

I wonder if we need an starting space, could you please check:
```suggestion
MACHINE_FEATURES += " efi ebbr"
```

---

No, in case of `+=` it doesn't need. The space is needed for `_append` operation.